services/capture/pom.xml

```diff
@@ -0,0 +1,234 @@
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>ca.bc.gov.tno</groupId>
+  <artifactId>capture</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+
+  <name>capture</name>
+  <description>TNO Media Capture Service</description>
+  <url>https://github.com/bcgov/tno</url>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+  <developers>
+    <developer>
+      <name>Stuart Morse</name>
+      <email>stuart.morse@quartech.com</email>
+      <organization>Quartech</organization>
+    </developer>
+    <developer>
+      <name>Jeremy Foster</name>
+      <email>jeremy.foster@fosol.ca</email>
+      <organization>Quartech</organization>
+    </developer>
+  </developers>
+  <scm>
+    <connection>scm:git:git@github.com:bcgov/tno.git</connection>
+    <developerConnection>scm:git:ssh://github.com:bcgov/tno.git</developerConnection>
+    <url>https://github.com/bcgov/tno.git</url>
+  </scm>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <java.version>17</java.version>
+    <maven.compiler.source>17</maven.compiler.source>
+    <maven.compiler.target>17</maven.compiler.target>
+    <jackson.version>2.13.1</jackson.version>
+    <kafka.version>3.0.0</kafka.version>
+  </properties>
+
+  <repositories>
+    <repository>
+      <id>maven-snapshots</id>
+      <name>Maven Central Repository - Snapshots</name>
+      <url>https://s01.oss.sonatype.org/content/repositories/snapshots/</url>
+    </repository>
+    <repository>
+      <id>maven-staging</id>
+      <name>Maven Central Repository - Staging</name>
+      <url>https://s01.oss.sonatype.org/content/groups/staging/</url>
+    </repository>
+    <repository>
+      <id>maven-releases</id>
+      <name>Maven Central Repository - Releases</name>
+      <url>https://s01.oss.sonatype.org/content/repositories/releases/</url>
+    </repository>
+    <repository>
+      <id>maven-central</id>
+      <name>Maven Central Repository</name>
+      <url>https://repo1.maven.org/maven2/</url>
+    </repository>
+  </repositories>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.6.3</version>
+  </parent>
+
+  <dependencies>
+    <dependency>
+      <groupId>ca.bc.gov.tno</groupId>
+      <artifactId>service</artifactId>
+      <version>0.0.9-SNAPSHOT</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.rometools</groupId>
+      <artifactId>rome</artifactId>
+      <version>1.18.0</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.jsoup</groupId>
+      <artifactId>jsoup</artifactId>
+      <version>1.14.3</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+      <version>2.17.1</version>
```

Comment:
Are these the latest?  Don't want log4j2

Reply:
This is the version they're pushing to address the exploit, although there is a newer one. I updated it to 2.17.2.